tools/jvm/src/main/scala/org/scalajs/core/tools/linker/backend/ClosureAstTransformer.scala

```diff
@@ -1,4 +1,4 @@
-package org.scalajs.core.tools.optimizer
+package org.scalajs.core.tools.linker.backend
```

Comment:
Should I move this into a subpackage just for closure?
Should I make this package private?